integration_tests/integration/serviceProviderReferrals.cy.js

```diff
@@ -1404,11 +1404,10 @@ describe('Service provider referrals dashboard', () => {
 
       const updatedAppointments = [appointmentWithSubmittedFeedback, appointments[1]]
       cy.stubGetActionPlanAppointments(actionPlan.id, updatedAppointments)
-
-      cy.contains('Return to service progress').click()
-
-      cy.get('[data-cy=session-table]')
+      cy.contains('Return to service progress').click().getTable()
+      cy.get('table[id="sessionAppointmentsTable"]')
```

Comment:
can we keep these as `cy.get('[data-cy=session-table]')` - getting by data attribute is considered best practice - https://docs.cypress.io/guides/references/best-practices#Selecting-Elements
(I've added a comment on interventionProgress.njk which would allow this)